dev/Breadcrumb/BreadcrumbBar.xaml

```diff
@@ -1,363 +1,8 @@
-                                        Visibility="Collapsed"
-                                        VerticalAlignment="Stretch"
-                                        IsTextScaleFactorEnabled="False"
-                                        FontSize="{TemplateBinding FontSize}"
-                                        AutomationProperties.AccessibilityView="Raw"
-                                        Grid.Column="0"/>
-
-                                <TextBlock x:Name="PART_ChevronTextBlock"
-                                        AutomationProperties.Name="ChevronTextBlock"
-                                        FontFamily="{ThemeResource SymbolThemeFontFamily}"
-                                        Foreground="{ThemeResource BreadcrumbBarNormalForegroundBrush}"
-                                        Text="&#xE76C;"
-                                        VerticalAlignment="Center"
-                                        HorizontalAlignment="Center"
-                                        IsTextScaleFactorEnabled="False"
-                                        FontSize="12"
-                                        AutomationProperties.AccessibilityView="Raw"
-                                        Padding="2,2,2,2"
-                                        Grid.Column="2"/>
-                            </Grid>
-                            
-                        </Button>
-                        <ContentPresenter x:Name="PART_EllipsisDropDownItemContentPresenter"
-                            x:DeferLoadStrategy="Lazy"
-                            Visibility="Collapsed"
-                            Content="{TemplateBinding Content}"
-                            ContentTemplate="{TemplateBinding ContentTemplate}"
-                            ContentTransitions="{TemplateBinding ContentTransitions}"
-                            HorizontalAlignment="Stretch"
-                            HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
-                            VerticalAlignment="{TemplateBinding VerticalContentAlignment}"/>                       
-                    </Grid>
-                </ControlTemplate>
-            </Setter.Value>
-        </Setter>
-    </Style>
-    
-    <Style TargetType="local:BreadcrumbBar">
-        <Setter Property="AutomationProperties.LandmarkType" Value="Navigation"/>
-        <Setter Property="Template">
-            <Setter.Value>
-                <ControlTemplate TargetType="local:BreadcrumbBar">
-                    <local:ItemsRepeater x:Name="PART_ItemsRepeater" />
-                </ControlTemplate>
-            </Setter.Value>
-        </Setter>
-    </Style>
+    xmlns:local="using:Microsoft.UI.Xaml.Controls">
 
+    <Style TargetType="local:BreadcrumbBar" BasedOn="{StaticResource DefaultBreadcrumbBarStyle}"/>
     <Style TargetType="local:BreadcrumbBarItem" BasedOn="{StaticResource DefaultBreadcrumbBarItemStyle}"/>
```

Comment:
please consolidate all styles into two files: generic.xaml and generic_v1.xaml
Also you need to update CustomTools because we don't need merge xaml for generic.xaml anymore.
At the same time, in XamlControlsResources, we don't need to load resource file like rs5_generic.xaml, but generic.xaml